drafts/draft-ietf-ippm-ioam-direct-export.xml

```diff
@@ -300,38 +305,98 @@ packets   |Encapsu-|     | Transit|     | Transit|     |Decapsu-|
      </figure>
 
 		  
-		  <t>The DEX option is used as a trigger to export IOAM data. The 
-		  trigger applies to transit nodes, the decapsulating node, and the 
-		  encapsulating node:</t>
+		  <t>The DEX option is used as a trigger to collect and/or export IOAM 
+		  data. The trigger applies to transit nodes, the decapsulating node, 
+		  and the encapsulating node:</t>
 		  
           <t><list style="symbols">
                       <t>An IOAM encapsulating node configured to incorporate 
-					  the DEX option encapsulates the packet with the DEX 
-					  option, and MAY export the requested IOAM data
-                      immediately. The IOAM encapsulating node is the only
-                      type of node allowed to push the DEX option.</t>
+					  the DEX option encapsulates (possibly a subset of) the 
+					  packets it forwards with the DEX option, and MAY export 
+					  and/or collect the requested IOAM data immediately. The 
+					  IOAM encapsulating node is the only type of node allowed 
+					  to push the DEX option.</t>
```

Comment:
s/The IOAM encapsulating node is the only type of node allowed to push the DEX option/Only IOAM encapsulating nodes are allowed to add the DEX Option-Type to a packet./